### ISSUE TITLE
Set boundaries based on path 

### ISSUE THREAD

**Issue by lovetrivedi** [react-grid-layout/react-draggable#312]
can we set boundaries to particular path like polygon type path
like =>
bounds={4,-3 -9,163 -12,462 236,494 859,444 1174,329 1397,1 769,3}

**Comment by STRML** [comment]
No. That would be quite complex. You could use draggablecore and calculate
the boundaries yourself.
